### DISCUSSION TITLE
Support only for premium?

### DISCUSSION THREAD

**Discussion by brobie1kanobi** [yapzhenyie/GadgetsMenu#835] [Q&A]
Hi, 
  I'm just wondering if the free version will get support for an issue or not? I have an issue (which will be reported if so) when connecting from server to server on my bungee.

  Issue: I have the "MySQL Inventory bridge" (MIB) plugin (which works as intended) connecting my hub and survival servers so you can take items from the hub (which has Gadgets installed on it) to the survival (which doesn't have Gadgets). When you go to the survival server and put an item in your slot the Gadgets Menu Item (GMI) would be in in your hotbar, then rejoin the hub, it attempts to give the GMI to you but then the MIB refreshes your inventory.

 Being that it drops the item on the ground (as described in #813) and then adds the GMI, THEN the MIB refreshes the inventory it is essentially creating a dupe glitch. I believe the suggestion in 813 would be a fix for this, but I am wondering what kind of support I will get (or if filing the issue is even worth my time).
 
 Any help or response will be greatly appreciated!
 ~brobie1kanobi
 
 Also, first post! 💪😄

**Comment by yapzhenyie** [comment]
Compatibility issue is hard to solve. I will try to figure out a solution to your problem.

**Reply by brobie1kanobi** [reply]
Understood! I appreciate the response and hopefully you can figure it out. I've been trying to work on a skript to resolve it, but to no avail yet.
